client/my-sites/plans/hooks/use-segmented-intent.tsx

```diff
@@ -23,16 +23,16 @@ export function useSegmentedIntent(
 	const surveyedIntent = data?.[ blogId ]?.[ 'what-brings-you-to-wordpress' ]?.[ 0 ];
 
 	if ( ! enabled || ! surveyedIntent || ! surveyedGoals ) {
-		return fallback;
+		return { segment: fallback, isFetchingSegment: isFetching };
```

Comment:
We should add another `if` before this, that handles `! enabled` and return `isFetchingSegment: false`.

Reply:
Updated